applications/StructuralMechanicsApplication/tests/solid_2p5d_test/solid_2p5d_parameters.json

```
@@ -0,0 +1,79 @@
+{
+    "problem_data"     : {
+        "problem_name"  : "solid_2p5d",
+        "parallel_type" : "OpenMP",
+        "echo_level"    : 0,
+        "start_time"    : 0.0,
+        "end_time"      : 1.0
+    },
+    "solver_settings"  : {
+        "solver_type"                     : "Static",
+        "model_part_name"                 : "Structure",
+        "domain_size"                     : 2,
+        "echo_level"                      : 0,
+        "analysis_type"                   : "non_linear",
+        "model_import_settings"           : {
+            "input_type"     : "mdpa",
+            "input_filename" : "solid_2p5d_test/solid_2p5d"
+        },
+        "material_import_settings"        : {
+            "materials_filename" : "solid_2p5d_test/solid_2p5d_materials.json"
+        },
+        "time_stepping"                   : {
+            "time_step" : 1.1
+        },
+        "line_search"                     : false,
+        "convergence_criterion"           : "and_criterion",
+        "displacement_relative_tolerance" : 0.0001,
+        "displacement_absolute_tolerance" : 1e-9,
+        "residual_relative_tolerance"     : 0.0001,
+        "residual_absolute_tolerance"     : 1e-9,
+        "max_iteration"                   : 10,
+        "rotation_dofs"                   : false,
+        "linear_solver_settings"       : {
+            "solver_type"         : "ExternalSolversApplication.super_lu"
```

```suggestion
            "solver_type"         : "EigenSolversApplication.sparse_lu"
```

this one is faster, plus the ExternalSolvers will be replaced by the EigenSolvers in the long run

---

But then we need to compile now the EigenSolversApplication ??

---

I would recommend so, it has more capabilities than the EigenSolversApp
We are slowly switching

Since ~1week the tests in Structural by default use the EigenSolversApp

---

actually you could remove the specification of the linear solver completely and use the default
Then it would also work locally if you don't have the EigenSolversApp compiled

---

Ok, I'll do that. The thing is that then it skips the test...

---

Ok thats true, if you run it locally without having the EigenSolversApp it will skip it